src/components/ImageView/Overlay/OverlayComponent.tsx

```diff
@@ -150,7 +150,7 @@ export class OverlayComponent extends React.Component<OverlayComponentProps> {
             }
 
             if (!settings.title.customText) {
-                currentStyleString += `, Title=${this.props.image?.store?.filename}`;
+                currentStyleString += `, Title=${this.props.image?.store?.filename.replaceAll(",", "_")}`;
```

Comment:
it seems % also causes the issue. We can use the image viewer settings - title to have a custom text for experimentation. If , or % is in the string, the WCS layout is gone. Interestingly, if , is the late char, it survives but it is not rendered.